lib/github-service.js

```diff
@@ -29,6 +29,30 @@ class GithubService {
     });
   }
 
+  async getAssociatedPullRequestNumberFromSha(){
```

Comment:
We should make this method private (outside of the class)
and call it like this : 
async getAssociatedPullRequestNumberFromSha(repository, commitSha){
    return await this.getOctokitInstance().request('GET /repos/{owner}/{repo}/commits/{commit_sha}/pulls', {
      owner: this.options.repo.split('/')[0],
      repo: this.options.repo.split('/')[1],
      commit_sha: this.options.refCommit,
    });

And call it within GithubService class : getAssociatedPullRequestNumberFromSha(this.option.repo, this.options.refCommit)

Reply:
Done